backend/code_review_backend/issues/tests/commands/test_cleanup_issues.py

```diff
@@ -62,7 +62,7 @@ def setUp(self):
                     head_repository=repo,
                 )
                 for i, repo in enumerate(
-                    (self.moz_central, self.autoland, self.test_repo)
+                    (self.moz_central, self.test_repo, self.autoland)
```

Comment:
1. This change causes the comments in line 52 and 53 to not reflect the code in line 69 anymore.
2. Because revision 2 and 3 both have 2 issues each, this change has no effect on test steps.

Reply:
You're right, I'll simply revert it